### PR TITLE
MBS-10235: Drop format when reusing mediums

### PR DESCRIPTION
**Pull request by reosarevok**
https://tickets.metabrainz.org/browse/MBS-10235

The medium format being kept when reusing a medium causes a lot of problems (when editors forget to unset it) yet it rarely is of any use: a tracklist being the same doesn't imply the format will be and in fact it's often the opposite, with the same release having CD, vinyl and digital versions, for example. This drops the format when reusing a medium, so that the user will have to either select a format or select the "I don't know" checkbox for it.

### CODE REVIEW

**Comment by reosarevok** [comment]
Where would you add these instructions? The release duplicates tab? (and does it really matter? the user will anyway see what gets copied soon after :) )

**Comment by yvanzo** [comment]
> Where would you add these instructions? The release duplicates tab?

On the line 93 of `tracklist.tt` I guess?

> (and does it really matter? the user will anyway see what gets copied soon after :) )

We could also relabel the button “Add Medium” as “Button” and let user see what happens ;)

**Comment by reosarevok** [comment]
Ok, added something, how does it look?

File: root/release/edit/tracklist.tt [hunk 1]
```diff
@@ -108,6 +108,7 @@
 
   <!-- ko if: searchResults() && searchResults().length > 0 -->
   <p>[% l('Select a medium from the search results below and then click "Add Medium".') %]</p>
+  <p>[% l('The tracklist for the medium (including the track/recording associations) will be loaded as a medium on this release.') %]</p>
```
Comment (yvanzo) [inline root/release/edit/tracklist.tt]: It should be made more clear that other medium’s attributes won’t be copied:
```suggestion
  <p>[% l('Only the tracklist for the medium (including the track/recording associations) will be loaded as a medium on this release.') %]</p>
```

Comment (reosarevok) [reply root/release/edit/tracklist.tt]: Modified the comment somewhat, let me know how it sounds.

Comment (yvanzo) [reply root/release/edit/tracklist.tt]: SGTM

**Pull request merged**